docs/node-operators/migration-guides/upgrade-to-0701.md

```diff
@@ -0,0 +1,235 @@
+```
+
+**Notes**: 
+* *The `tree` command needs to be installed (e.g. `apt-get install -y tree`) but it is the easiest way to see if backup files match the original files without going too deep into details.
+*  **You might see some errors when running `pg_dump`. To learn if they can be safely ignored, see the [troubleshooting section in the official timescaledb docs ↗](https://docs.timescale.com/timescaledb/latest/how-to-guides/backup-and-restore/troubleshooting/).
+
+
+### 3. Download new binaries
+Download new binaries for the upgrade version from the [GitHub releases page ↗](https://github.com/vegaprotocol/vega/releases) and unzip them. Save them in the `<VEGA-BIN>` and the `<VISOR-BIN>` you chose. 
+
+The binaries you need: 
+* Vega binary: Also includes Tendermint and data node as binary subcommands
+* Visor binary: Optional for setting up Visor for protocol upgrades (See the docs listed in [step 7](#7-read-the-visor-documentation) for information on Visor.
+
+See example commands for downloading below. You may need to update the version number depending on the version of the binaries you need to update to:
+
+```bash
+# Download archives
+wget https://github.com/vegaprotocol/vega/releases/download/v0.70.1/vega-linux-amd64.zip
+wget https://github.com/vegaprotocol/vega/releases/download/v0.70.1/visor-linux-amd64.zip
+
+# Unzip downloaded archives
+unzip vega-linux-amd64.zip
+unzip visor-linux-amd64.zip
+
+mv vega <VEGA-BIN>
+mv visor <VISOR-BIN>
+```
+
+:::node  Manual update for binaries when you are running visor
+Visor cannot automatically restart(or upgrade) binaries because We are restarting the network with the `checkpoint`. Once you stop the Vega network, you must update the old binaries with the downloaded ones. You can find the binary path in the `<VEGAVISOR-HOME>/current/run-config.toml` file. Usually the path is `<VEGAVISOR-HOME>/current/vega`
+:::
+
+### 4. Reset and clear all data
+
+:::warning Back up files before progressing
+Ensure you have a backup of the network files because the steps below will remove data from your home.
+
+You may also risk losing your wallets, so back them up as well.
+:::
+
+1. Call unsafe reset all for Tendermint: `<VEGA-BIN> tendermint unsafe-reset-all --home <TENDERMINT-HOME>`
+2. Call unsafe reset all for vega core: `<VEGA-BIN> unsafe_reset_all --home <VEGA-NETWORK-HOME>`
+3. Remove the data-node state directory: `rm -f <VEGA-NETWORK-HOME>/state/data-node`
+4. Recreate the PostgreSQL database if you have data within: 
+    - a. Call the following command in PostgreSQL terminal: `DROP DATABASE IF EXISTS <VEGA-DB-NAME>`
+    - b. Call the following command in PostgreSQL terminal: `CREATE DATABASE <VEGA-DB-NAME> WITH owner=<VEGA-DB-USER>`. 
+
+:::note
+If you have to setup postgresql, follow instructions in the step to [Install/Upgrade PostgreSQL instance](./upgrade-node.md#15-installupgrade-postgresql-for-data-node)
+:::
+

```

Comment:
Should this be removed? There is no step 15/info on how to do this

Reply:
The step 15 is in previous migration guide